base_to_camera_guess:
  x: 1.0  # Note: Until I fix this YOU NEED to make this a DOUBLE! Don't write just '0'.
  y: 1.0
  z: 0.5
  qx: 0.0
  qy: -0.7071068
  qz: 0.7071068
  qw: 0.0
wrist_to_target_guess:
  x: 0.0
  y: 0.0
  z: 0.0
  qx: -0.7071068
  qy: 0.0
  qz: 0.0
  qw: 0.7071068
